client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -378,26 +365,31 @@ const canCurrentUserFn = ( state, siteId ) => capability =>
  * A functional selector similar to `canCurrentUserFn`, this time for generating editor URL
  * from a post type.
  */
-const getNewPostPathFn = ( state, siteId ) => postTypeSlug =>
-	getEditorNewPostPath( state, siteId, postTypeSlug );
+const getEditorUrlFn = ( state, siteId ) => postType =>
```

Comment:
🤔 Maybe `getEditorUrlForPostType`. Not loving the Fn postfix, but just a nit.

Reply:
Hmm looking at this wrapper is there any reason why postType isn't available in mapStateToProps? Not sure we need this.

https://lodash.com/docs/#partial may be of interest as well.

Reply:
👍  to avoid using `Fn`. As @gwwar mentions, you can do something like:

```js
import { getEditorUrl as getEditorUrlStateSelector } from 'state/selectors/get-editor-url';
```

and then on `mapStateToProps`:

```js
getEditorUrl: partial( getEditorUrlStateSelector, state, siteId, null ),
```

avoiding the function altogether.

`postType` isn't available because this is for the sidebar menu, where we need to create the links for all the post types.